source/adapters/level_zero/v2/kernel.cpp

```diff
@@ -391,16 +391,14 @@ ur_result_t urKernelGetGroupInfo(
     kernelProperties.pNext = &workGroupProperties;
 
     auto zeDevice = hKernel->getZeHandle(hDevice);
-    if (zeDevice) {
-      auto zeResult =
-          ZE_CALL_NOCHECK(zeKernelGetProperties, (zeDevice, &kernelProperties));
-      if (zeResult == ZE_RESULT_SUCCESS &&
-          workGroupProperties.maxGroupSize != 0) {
-        return returnValue(workGroupProperties.maxGroupSize);
-      }
-      return returnValue(
-          uint64_t{hDevice->ZeDeviceComputeProperties->maxTotalGroupSize});
+    auto zeResult =
```

Comment:
As far as I can tell, kHernel->getZeHandle should never return null. @oneapi-src/unified-runtime-level-zero-v2-write , am I correct in thinking this?

Reply:
yes